lib/api/protocol/frame.js

```diff
@@ -31,15 +31,8 @@ module.exports = class Session extends ProtocolAction {
       return this.transportActions.switchToFrame(callback);
     } 
     
-    if (arguments.length === 2 && isString(frameId)){ 
-      try {
-        frameId = await this.driver.findElement(By.css(`#${frameId}, *[name="${frameId}"]`));
-      } catch (err){
-        return {
-          status: -1,
-          error: err
-        };
-      }
+    if ([1, 2].includes(arguments.length) && isString(frameId)){ 
+      frameId = await this.driver.findElement(By.css(`#${frameId}, *[name="${frameId}"]`));
```

Comment:
This is breaking the behavior of the frame command. Frame command should not terminate the execution upon failure.